### PR TITLE
[ios][sdk36] Add React-RCTCameraRoll to SDK36 codebase

### PR DESCRIPTION
**Pull request by tsapeta**
# Why

Part of iOS Quality Assurance process (#6264). CameraRoll is no longer included within React Native which causes that there is no appropriate request handler/image loader to load images with `assets-library://` url scheme which come from user's media library. 

# How

Added temporary `ABI36_0_0React-RCTCameraRoll.podspec` to the codebase versioned for SDK36 and installed this pod.

*I've decided not to fix this issue in `UNVERSIONED` environment as we're planning to come up with a better solution for this problem in the next SDK.*

# Test Plan

- Set `sdkVersion` in `apps/native-component-list/app.json` to `36.0.0`.
- Start `native-component-list` and open up `MediaLibrary` example - it should show photos from your user's library.

### CODE REVIEW

File: ios/versioned-react-native/ABI36_0_0/ReactNative/autolink-ios.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@ def use_react_native_ABI36_0_0! (options={})
   pod 'ABI36_0_0React-RCTActionSheet', :path => "#{prefix}/Libraries/ActionSheetIOS", :project_name => 'ABI36_0_0'
   pod 'ABI36_0_0React-RCTAnimation', :path => "#{prefix}/Libraries/NativeAnimation", :project_name => 'ABI36_0_0'
   pod 'ABI36_0_0React-RCTBlob', :path => "#{prefix}/Libraries/Blob", :project_name => 'ABI36_0_0'
+  pod 'ABI36_0_0React-RCTCameraRoll', :path => "#{prefix}/Libraries/CameraRoll", :project_name => 'ABI36_0_0'
```
Comment (sjchmiela) [inline ios/versioned-react-native/ABI36_0_0/ReactNative/autolink-ios.rb]: Should we add some small comment here that it has been created manually by you?

**Review by sjchmiela — APPROVED** [review]
LGTM apart from this one comment 🚀

**Review by esamelson — APPROVED** [review]
makes sense, thanks! could you open an issue to track a more permanent fix in UNVERSIONED?

**Comment by rxhivzero** [comment]
@tsapeta hello
Same problem in my SDK 36 expokit, is there any way I can fix it?

**Comment by rxhivzero** [comment]
https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz
Latest version now Same problem.

**Comment by tsapeta** [comment]
Hey @rxhivzero,
Yeah, sorry for this problem, the solution I made here wasn't ideal and we will come up with a better one in the next SDK. For now, if you're ejected to ExpoKit or bare workflow then you can just install https://github.com/react-native-community/react-native-cameraroll which includes [RNCAssetsLibraryRequestHandler](https://github.com/react-native-community/react-native-cameraroll/blob/master/ios/RNCAssetsLibraryRequestHandler.m) that is needed to properly handle `assets-library://` image source urls. Let me know if it doesn't help!

**Comment by rxhivzero** [comment]
@tsapeta 
ok this work.

**Pull request merged**